### DISCUSSION TITLE
Not able to use starship

### DISCUSSION THREAD

**Discussion by Brajesh3** [Xed-Editor/Xed-Editor#130] [General]
I installed ***starship*** by using command `apk add starship`
My default shell is bash
After i set `eval "$(starship init bash)"` in `~/.bashrc`
& when i launch terminal it freezes i have to exit it with `ctrl + c`

![Screenshot_20240909-235113_Karbon](https://github.com/user-attachments/assets/f3892a5a-0640-423e-8bba-284b9938d878)

**Comment by RohitKushvaha01** [comment]
> I installed ***starship*** by using command `apk add starship`
> My default shell is bash
> After i set `eval "$(starship init bash)"` in `~/.bashrc`
> & when i launch terminal it freezes i have to exit it with `ctrl + c`
> 
> ![Screenshot_20240909-235113_Karbon](https://github.com/user-attachments/assets/f3892a5a-0640-423e-8bba-284b9938d878)
> 

1. clear app data
2. open terminal
3.
 ```bash
  apk add starship
  echo 'eval "$(starship init bash)"' | tee -a /etc/bash/bashrc > /dev/null
 ```

4. Run bash
